UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/CommonHelperTest.cs

```diff
@@ -70,5 +87,81 @@ public void CheckGetFileSize()
             long fileSize = CommonHelper.GetFileSize(salesCatalogueResponse.ResponseBody);
             Assert.AreEqual(500, fileSize);
         }
+
+        [Test]
+        public async Task GetRetryPolicyWhenTooManyRequests()
```

Comment:
The naming of this test is inconsistent with earlier naming conventions i.e. WhatWeExpect_WhenWeDoSomething

Reply:
Yes. Updated accordingly